src/main/webapp/registration.js

```diff
@@ -57,23 +57,32 @@ function displayLoginLogoutLinkHelper(document, loginStatus) {
     if (loginStatus.isLoggedIn) {
         document.getElementById('login').style.display = "none";
         document.getElementById('profile').style.display = "block";
-        document.getElementById('availability-settings').style.display = "block";
-        document.getElementById('tutor-session-settings').style.display = "block";
-        document.getElementById('history').style.display = "block";
         document.getElementById('logout').style.display = "block";
         document.getElementById('logout-url').href = loginStatus.url;
         document.getElementById('profile').addEventListener('click', () => {
             setProfileQueryString(window, loginStatus);
         });
-        document.getElementById('availability-settings').addEventListener('click', () => {
-            redirectToManageAvailability(window, loginStatus);
-        });
-        document.getElementById('tutor-session-settings').addEventListener('click', () => {
-            redirectToManageSessions(window, loginStatus);
-        });
-        document.getElementById('history').addEventListener('click', () => {
-            redirectToHistory(window, loginStatus);
-        });
+
+        // If the user is a tutor, display availability settings
+        if (loginStatus.tutor) {
```

Comment:
What happens here if the user is both a tutor and a student?

Reply:
After I implement the possibility of registering as both student and tutor, there will be a tab to see the student's view and the tutor's view. But since allowing the student to register as both student and tutor would involve making changes to the registration servlet and others, I'm doing that in a different PR just so that it is more organized.

Reply:
Sounds good!